components-page/components.html

```diff
@@ -0,0 +1,168 @@
+            </div>
+
+            <div class="item">
+              <label for="bookmark" class="item-text">Modal</label>
+            </div>
+
+            <div class="item">
+              <label for="bookmark" class="item-text">Navigation</label>
+            </div>
+          </div>
+        </aside>
+        <div class="components-items">
+          
+          <div class="buttons">
+            <div class="heading btn-heading">Primary & Secondary Button</div>
+            <div class="btns">
+              <div class="btn-basic">
+                <button class="btn btn-primary">Primary</button>
+                <button class="btn btn-secondary">Secondary</button>
+              </div>
+              <div class="btn-code code">
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%28138%2C43%2C226%2C1%29&t=night-owl&wt=none&l=htmlmixed&width=621&ds=true&dsyoff=0px&dsblur=33px&wc=true&wa=true&pv=50px&ph=59px&ln=false&fl=1&fm=ReadexPro-Regular&fs=13px&lh=145%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EPrimary%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253ESecondary%253C%252Fbutton%253E%250A"
+                  style="
+                    width: 507px;
+                    height: 231px;
+                    border: 0;
+                    transform: scale(1);
+                    overflow: hidden;
+                  "
+                  sandbox="allow-scripts allow-same-origin"
+                >
+                </iframe>
+              </div>
+            </div>
+            <div class="heading btn-heading">Link Button</div>
+            <div class="btns">
+              <button class="btn btn-link"><a href="#">Link 1</a></button>
+              <button class="btn btn-link"><a href="#">Link 2</a></button>
+              <div class="btn-code code">
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%28138%2C43%2C226%2C1%29&t=night-owl&wt=none&l=htmlmixed&width=621&ds=true&dsyoff=0px&dsblur=33px&wc=true&wa=true&pv=50px&ph=59px&ln=false&fl=1&fm=ReadexPro-Regular&fs=13px&lh=145%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-link%2522%253E%253Ca%2520href%253D%2522%2523%2522%253ELink%25201%253C%252Fa%253E%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-link%2522%253E%253Ca%2520href%253D%2522%2523%2522%253ELink%25202%253C%252Fa%253E%253C%252Fbutton%253E%250A"
+                  style="
+                    width: 539px;
+                    height: 231px;
+                    border: 0;
+                    transform: scale(1);
+                    overflow: hidden;
+                  "
+                  sandbox="allow-scripts allow-same-origin"
+                >
+                </iframe>
+              </div>
+            </div>
+            <div class="heading btn-heading">Icons Button</div>
+            <div class="btns">
```

Comment:
this `div` is a **container** for buttons so you can use class name like `btn-container` or `btn-wrapper`